web/src/installer/components/store.js

```diff
@@ -28,8 +28,8 @@ export const StepEnum = {
   USER: 'user',
 }
 
-const defaultServiceSubnet = '10.100.0.0/16';
-const defaultPodSubnet = '10.244.0.0/16';
+const defaultServiceSubnet = '100.100.0.0/16';
+const defaultPodSubnet = '100.96.0.0/16';
```

Comment:
Great job on UI side :)

Could you also change the placeholders of input fields in here https://github.com/gravitational/gravity/blob/master/web/src/installer/components/StepProvider/AdvancedOptions/Subnets/Subnets.jsx#L43

Reply:
Should we also update the docs as well?

https://github.com/gravitational/gravity/blob/master/docs/6.x/installation.md#L115

> Must be a minimum of /16 so Kubernetes is able to allocate /24 to each node. Defaults to 10.244.0.0/16

Reply:
I wasn't planning on backporting this, so until we separate out the 7.x docs I don't think we should change them.

Reply:
Yeah, I guess what I was trying to say that it needs to be updated in the docs too. I would create an issue so we do not forget to make this change in docs before 7.0 release.